### DISCUSSION TITLE
¿Puedo agregar yo mi carpeta?

### DISCUSSION THREAD

**Discussion by JLiliam** [Taller-Abierto-de-Humanidades-Digitales/curso_python_extenso#26] [Dudas del curso]
Profe Jairo:
Por fin pude crear mi _branch_. ¿Puedo crear mi carpeta en la de estudiantes o es algo que debe haceer el profesor? 

¡Gracias!
Jeniffer Liliam Mendoza

**Comment by jairomelo** [comment]
Hola Jennifer,
¡Súper! Qué bueno que ya hiciste tu branch :)

Sí, puedes crear tu carpeta estudiante y dentro de ella crea un archivo README.md con tus datos (como está indicado en la actividad1)

Muchos saludos
